applications/crossbar/src/modules/cb_accounts.erl

```diff
@@ -261,6 +265,17 @@ post(Context, AccountId, ?MOVE) ->
         _Status -> Context
     end.
 
+-spec patch(cb_context:context(), path_token()) -> cb_context:context().
+patch(Context, _AccountId) ->
+    Context1 = crossbar_doc:save(Context),
+    case cb_context:resp_status(Context1) of
+        'success' ->
+           _ = provisioner_util:maybe_update_account(Context1);
```

Comment:
This clause needs to return a `cb_context:context()`, also `provisioner_util` functions should run inside their own process, [like in cb_accounts](https://github.com/2600hz/kazoo/blob/master/applications/crossbar/src/modules/cb_accounts.erl#L248)